src/modules/editor/utils.spec.ts

```diff
@@ -88,3 +89,66 @@ describe('when converting a catalyst wearable to a wearable', () => {
     expect(fromCatalystWearableToWearable(aCatalystWearable)).toEqual(aWearable)
   })
 })
+
+describe('when patching wearables', () => {
+  describe("and the category of the wearable is 'skin'", () => {
```

Comment:
Would you mind adding the other case? The `and the category of the wearable isn't a 'skin'`

Reply:
Done ✅